src/clndr.js

```diff
@@ -205,10 +211,11 @@
                 if (this.options.lengthOfTime.startDate) {
                     this.intervalStart =
                         moment(this.options.lengthOfTime.startDate)
-                            .startOf('day');
+                            .startOf('week');
```

Comment:
This breaks a use case that I think is valid and useful... Using `lengthOfTime.days` you should be able to create a calendar that spans an arbitrary number of days (2, 3, 5, etc.). This hardcodes the assumption that the user wants a calendar specified in _weeks_, and now it will not honor the `lengthOfTime.startDate` option (instead going to the beginning of that week).

I tested this by making a 3 day calendar and setting the `lengthOfTime.startDate` to the middle of the week. It starts on the previous Sunday instead.

Does that make sense? Am I missing anything?